pro/src/components/pages/Home/Offerers/InvalidBusinessUnits.jsx

```diff
@@ -22,6 +22,7 @@ const InvalidBusinessUnits = ({ offererId, hasTitle = true }) => {
 
       <div className="h-card-content">
         <InternalBanner
+          minimalStyle={true}
```

Comment:
Détail de style/goût, je péfrère écrire : 
```suggestion
        <InternalBanner
          minimalStyle
```